docs/evidence/change-aware-ci-proof-of-life.md

```
@@ -0,0 +1,295 @@
+- Run ID: **PENDING**
+- `selection.json` content:
+
+  ```json
+  PENDING
+  ```
+
+- Unit shard count: **PENDING** (expected: 40)
+
+### Branch-Protection Contexts
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)